usaspending_api/references/v2/views/city.py

```diff
@@ -74,7 +74,7 @@ def create_elasticsearch_query(return_fields, scope, search_text, country, state
     # so that we don't get inconsistent results when the limit gets down to a very low number (e.g. lower than the
     # number of shards we have) such that it may provide inconsistent results in repeated queries
     city_buckets = limit + 100
-    if country == ALL_FOREIGN_COUNTRIES:
```

Comment:
Is this variable used elsewhere? If not, we can go ahead and remove it. If so, why is the logic different in different places? The way I see it, either anything that isn't "USA" is foreign, OR we want to hold out for the sentinel value.

Reply:
For a specific foreign country, we use the actual country code, like 'GBR' for the United Kingdom. In this case, we want to return the country code for both ALL_FOREIGN_COUNTRIES as well as for individual foreign countries, like 'GBR'. Thus, anything not 'USA', where we return the state codes instead,